docs/rag/adrs/adr-vectordb.md

```diff
@@ -0,0 +1,26 @@
+# Initial InstructLab Vector Store
+
+## Context
+
+One of the first choices to make in implementing RAG is to choose an initial vector store to develop against. Though the usage of frameworks like LangChain or Haystack make it easy to swap vector databases, we need a working end to end implementation for RAG that is tested against and available to install with InstructLab. There are many options (see [here](https://docs.haystack.deepset.ai/docs/choosing-a-document-store)). 
```

Comment:
How do I disable the below linting bot comment? This is a useless nitpick. @nathan-weinberg maybe you can help here?